charts/tidb-cluster/templates/tidb-cluster.yaml

```diff
@@ -18,6 +18,13 @@ metadata:
 {{ toYaml .Values.extraLabels | indent 4 }}
     {{- end }}
 spec:
+  {{- if .Values.tcpKeepaliveTime }}
+  tcpKeepaliveTime: {{ .Values.tcpKeepaliveTime }}
+  tcpKeepaliveIntvl: 75
+  {{- end }}
+  {{- if .Values.soMaxConn }}
+  soMaxConn: {{ .Values.soMaxConn }}
```

Comment:
Putting these parameters at cluster level is a little misleading, how about moving these to component? e.g. `.sepc.tidb.tcpKeepaliveTime`, `.spec.tikv.soMaxConn`

Reply:
I prefer a general solution. How about using [the sysctl struct from Kubernetes](https://github.com/kubernetes/api/blob/25240d7d6d90fc391c9f53db9d620a313264d5d5/core/v1/types.go#L3184)?
 
It's an alternative way when configuring via Kubelet is not avaiable, no need to use another structure.

Reply:
+1 config per component, like podSecurityContext which we are using now

Reply:
Reuse the PodSecurityContext in each component.